user/getting-started.md

```diff
@@ -1,4 +1,35 @@
 
 # Getting Started
 
-* **TODO**
\ No newline at end of file
+Right now, 2017Q2 the easiest way to get started is to use local driver development environment: manually clone and build a container for a language driver with an SDK, then run it through Docker in order to get source -> UAST.
+
+I.e for a [Java driver](https://github.com/bblfsh/java-driver) :
+
+## Pre-requests
+ - Docker
+ - Go, for SKD build
+ - (macOS) `coreutils` and `gettext`, for SDK to work
```

Comment:
I believe OS used to be called "OS X", but since last release it's official name going further is [macOS](https://en.wikipedia.org/wiki/Macintosh_operating_systems#Mac_OS_X.2FOS_X.2FmacOS) and that's why it was used. 

Should use the old one 👎  or a new one 👍  ?